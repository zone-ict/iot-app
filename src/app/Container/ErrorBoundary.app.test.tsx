import React from 'react';
import renderer from 'react-test-renderer';
import ErrorBoundary from './ErrorBoundary.app';

describe('Error boundary component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <ErrorBoundary>
          <div>Test Content</div>
        </ErrorBoundary>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
