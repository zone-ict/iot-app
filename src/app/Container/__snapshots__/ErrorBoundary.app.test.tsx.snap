// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error boundary component should render correctly 1`] = `
<div>
  Test Content
</div>
`;
